src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
```

Comment:
It's better to select the table body (`tbody`) at the beginning of the script, similar to how you've selected the table head (`thead`). This will allow you to avoid querying the DOM each time the `click` event is triggered, which can be a performance issue in a large table.